### ISSUE TITLE
Disable Internal Trackpad?

### ISSUE THREAD

**Issue by ssmith353** [syscl/M3800#39]
Hey @syscl I'm trying to disable the internal trackpad when I have an external USB mouse, any ideas on how to do that? 

The System Preferences -> Accessibility -> Mouse & Trackpad -> "Ignore when external mouse is present" doesn't have any effect. 

Any ideas?

**Comment by syscl** [comment]
I may write a kext/program to try to fix what u mentioned above. But since the trackpad is regconized as mouse under Mac, maybe it will be a tough job. But I will try later.

syscl
